hack/dockerfile/install/containerd.installer

```diff
@@ -27,10 +28,9 @@ install_containerd() {
 		make
 	)
 
-	mkdir -p "${PREFIX}"
-
-	cp bin/containerd "${PREFIX}/containerd"
-	cp bin/containerd-shim "${PREFIX}/containerd-shim"
-	cp bin/containerd-shim-runc-v2 "${PREFIX}/containerd-shim-runc-v2"
-	cp bin/ctr "${PREFIX}/ctr"
-}
+	install -D bin/containerd "${PREFIX}/containerd"
+	install -D bin/containerd-shim "${PREFIX}/containerd-shim"
+	install -D bin/containerd-shim-runc-v1 "${PREFIX}/containerd-shim-runc-v1"
```

Comment:
unused and deprecated

Reply:
Ah, I can remove it; I added it because since https://github.com/docker/containerd-packaging/pull/146, it would also be included in the deb/rpm packages

Should we remove it from the containerd makefile (default) target?

Reply:
containerd-packaging should continue packaging `containerd-shim-runc-v1`